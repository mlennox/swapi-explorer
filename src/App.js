import React, { Fragment } from "react";
import Nav from "./nav/nav";

const App = () => (
  <Fragment>
    <Nav />
  </Fragment>
);

export default App;
