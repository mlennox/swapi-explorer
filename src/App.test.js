import React from "react";
import App from "./App";
import { render, cleanup } from "react-testing-library";
import "jest-dom/extend-expect";

afterEach(cleanup);

describe("App", () => {
  test("it renders without crashing", () => {
    const { container } = render(<App />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
