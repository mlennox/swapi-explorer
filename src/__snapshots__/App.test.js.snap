// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App it renders without crashing 1`] = `
<nav>
  <a
    aria-current="page"
    class="active"
    href="/"
  >
    Home
  </a>
  <a
    href="/people"
  >
    People
  </a>
  <a
    href="/planets"
  >
    Planets
  </a>
  <a
    href="/species"
  >
    Species
  </a>
  <a
    href="/ships"
  >
    Ships
  </a>
  <a
    href="/vehicles"
  >
    Vehicles
  </a>
  <a
    href="/films"
  >
    Films
  </a>
</nav>
`;
